src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -148,11 +167,12 @@
     <Reference Include="Microsoft.CSharp" />
     <Reference Include="System.Xml" />
     <Reference Include="WindowsBase" />
-	<PackageReference Include="Extended.Wpf.Toolkit" version="3.0.0">
+	<PackageReference Include="Extended.Wpf.Toolkit" version="4.5.0">
 	  <GeneratePathProperty>true</GeneratePathProperty>
     <!--Exclude copying the dll because we will handle it in the BinaryCompatibilityOps target -->
     <ExcludeAssets>runtime</ExcludeAssets>
 	</PackageReference>
+      <FrameworkReference Condition=" '$(TargetFramework)' == 'net6.0-windows' " Include="Microsoft.WindowsDesktop.App" />
```

Comment:
is this needed for WPF - @pinzart90, any info on this?

Reply:
Fixed. I removed the line. It wasn't necessary.